content/en/logs/log_collection/java.md

```diff
@@ -65,18 +65,21 @@ Add a new file appender to `log4j.xml`:
 </appender>
 ```
 
-**Inject trace IDs in your logs**
+## Connect your service across Logs and APM
 
-If APM is enabled for this application and you wish to improve the correlation between application logs and traces, [follow these instructions][1] to set the [MDC (Mapped Diagnostic Contexts)][2] to then automatically add trace and span id in your logs.
+If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
+See [APM Java logging instructions][1] to learn more about adding trace IDs, span IDs, and even the
+`env`, `service`, and `version` to your logs.
+
+**Note**: If the APM tracer injects `service` into your logs, there is no need to set a `service` field in your logs configuration.
 
 Once this is done, the `ConversionPattern` to use becomes:
 
 ```xml
-<param name="ConversionPattern" value="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %X{dd.trace_id} %X{dd.span_id} - %m%n" />
+<param name="ConversionPattern" value="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %X{dd.env} %X{dd.service} %X{dd.version} %X{dd.trace_id} %X{dd.span_id} - %m%n" />
```

Comment:
@PierreVanRoy @palazzem let's first test this and maybe take this opportunity to restore the keyvalue format with quotes for Java logs as well.